### ISSUE TITLE
Bij het verwijderen van klanten lijkt het erop hij al gedisabled wordt voordat er bevestigd wordt. rara

### ISSUE THREAD

**Issue by jtborger** [deregenboog/ecd#1615]
zelf gezien...
:)

**Comment by barthuttinga** [comment]
@jtborger Ik kan dit niet reproduceren in dev env. Heb je meer info?

**Comment by jtborger** [comment]
Nee... ik weet het niet meer en ik zie dat het gewoon werkt zoals het zou moeten. geen flauw idee. ik sluit hem...

**Comment by dvleeuwen** [comment]
Ha Jan en Bart, volgens mij was dit het probleem bij het verwijderen van deelnemer en het "tussenscherm":
Als je iemand verwijdert krijg je eerst de vraag wil je dit zeker, of bericht "dit kan niet want er staat nog wat open" ofzo.
Maar dan is de klant ondertussen toch al wel verwijderd. 
Althans als ik het me goed herinner...

**Comment by jtborger** [comment]
Bij de vraag 'Weet u het zeker', gaat het wel goed (althans, dat kunnen we niet reproduceren). Bij die melding: kan niet want er staat nog wat open, gaat het denk ik mis... dat moeten we even onderzoeken. Thanks!

**Comment by dvleeuwen** [comment]
ah ja kan goed kloppen dat het enkel bij die waarschuwing van "kan niet verwijderen want staat nog wat open" is idd.

daarnaast krijg je volgens mij ook nog altijd een soort foutmelding te zien nadat je een klant verwijderd hebt...

**Comment by toomba-farhad** [comment]
Ik heb geprobeerd het probleem te reproduceren. Ik heb geprobeerd om afhankelijkheden met andere tabellen te creëren, maar ik kon de gerapporteerde fout niet reproduceren.

**Comment by jtborger** [comment]
Ik kon het ook niet meer reproduceren, dus we sluiten hem. Tnx!
